src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -23,15 +49,29 @@
 {% set _css_class = 'ecl-banner' %}
 {% set _extra_attributes = 'data-ecl-banner data-ecl-auto-init="Banner"' %}
 {% set _picture = picture|default({}) %}
-{% set _variant = variant|default('plain-background') %}
+{% set _video = video|default({}) %}
 {% set _size = size|default('m') %}
-{% set _centered = centered ?? false %}
+{% set _font_size = font_size|default('m') %}
```

Comment:
wow this is what they want when a user chooses the "m" font..? 
![image](https://github.com/ec-europa/europa-component-library/assets/14326885/34ea9240-882a-458d-8050-b1f7c93892a9)
ugly..